source/mongodb-handshake/handshake.rst

```diff
@@ -96,6 +100,50 @@ disconnect.
 MUST NOT contain handshake arguments. Any subsequent ``hello`` or legacy hello calls,
 such as the ones for topology monitoring purposes, MUST NOT include this argument.
 
+Example Implementation
+~~~~~~~~~~~~~~~~~~~~~~
+
+Consider the following pseudo-code for establishing a new connection:
+
+.. code:: python
+
+ conn = Connection()
+ conn.connect()  # Connect via TCP / TLS
+ if versioned_api_configured:
```

Comment:
What about load balanced mode?

Reply:
Done.

Reply:
This work has been moved to a separate ticket: https://jira.mongodb.org/browse/DRIVERS-2033